internal/indexmigration/migration.go

```diff
@@ -48,15 +44,22 @@ func Migrate(paths environment.Paths) error {
 		klog.V(2).Infoln("Already migrated")
 		return nil
 	}
+	indexPath := paths.IndexPath()
+	tmpPath := filepath.Join(paths.BasePath(), "tmp_index_migration")
```

Comment:
this wouldn't be better as os.TempDir?

Reply:
TempDir would be safer of course, but I wanted to avoid moving across file systems.